src/styles/blocks/container.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  display: flex;
+  position: absolute;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.